kyuubi-server/web-ui/src/components.d.ts

```diff
@@ -0,0 +1,24 @@
+// generated by unplugin-vue-components
```

Comment:
can we add license header on this file as well? or if it's auto generate by build command, we can exclude it from git

Reply:
No, I tried, I can try if I can hard code it

Reply:
ditto, could it be removed now?